client/src/controllers/KeyboardController.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus';
+import Mousetrap from 'mousetrap';
+
+export class KeyboardController extends Controller<HTMLButtonElement> {
+  static values = { key: String };
+
+  handleKey(event: Event) {
+    event.preventDefault();
+    this.element.click();
+  }
+
+  initialize(): void {
```

Comment:
```suggestion
  initialize() {
```

No need for the `void` return, that's already implied and TypeScript will do that for us.

Also I suggest you move `initialize` above the `handelKey`, it's easier to follow the code if it's near the top.